### ISSUE TITLE
java API had no bool tensorflow，how to add it to the session in java

### ISSUE THREAD

**Issue by lingbao00** [tensorflow/tensorflow#13601]
i am  doing transer the facenet to android,the input is the img,and the phase_train is a bool data,but the java api had no bool to be feed to session,as i had down this:

```
    private String inputName;
    private String phaseName;
    private String outputName;
    private int inputSize;

    private int[] intValues;
    private int[] valuedata;
    private float[] outputs;

 bitmap.getPixels(intValues, 0, bitmap.getWidth(), 0, 0, bitmap.getWidth(), bitmap.getHeight());
        for(int i = 0; i <intValues.length(); i++ )
        {
            final int val = intValues[i];
            valuedata[i * 3 + 0] = ((val >> 16) & 0xFF);
            valuedata[i * 3 + 1] = ((val >> 8) & 0xFF);
            valuedata[i * 3 + 2] = (val & 0xFF);
        }
inferenceInterface.fillNodeInt(
                inputName, new int[]{1, inputSize, inputSize, 3}, valuedata);
```
and then the c++ write :
m_phase_tensor = tensorflow::Tensor(tensorflow::DT_BOOL, tensorflow::TensorShape());
m_phase_tensor.scalar<bool>()() = false;
how to writen in java,can somebody help me!

**Comment by asimshankar** [comment]
The Java API supports boolean tensors but the `TensorFlowInferenceInterface` class does not do so yet. See: https://stackoverflow.com/questions/43001532/how-to-feed-boolean-placeholder-by-means-of-tensorflowinferenceinterface-java 

Contributions are welcome to make the change suggested in that StackOverflow answer.

FYI @andrewharp

**Comment by yongtang** [comment]
I think this issue is resolved with PR #14059 merged. Maybe the issues could be closed?

**Comment by asimshankar** [comment]
Yup, thanks.
